src/components/EventManagement/AddNewEvents.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import { eventPostData } from "../../api";
+
+const AddNewEvents = () => {
+  const [title, setTitle] = useState("");
+  const [weddingDecription, setWeddingDecription] = useState("");
+  const [startTime, setStartTime] = useState(new Date());
+  const [endTime, setEndTime] = useState(new Date());
```

Comment:
use date-fns plugin for date implementation

Reply:
done